frontend/src/components/archive/ArchiveErrorBoundary.tsx

```
@@ -0,0 +1,193 @@
+            {/* Action buttons */}
+            <div className="flex gap-3 mt-4">
+              <Link
+                to="/archives"
+                className="px-4 py-2 bg-blue-600 text-white rounded hover:bg-blue-700 transition-colors font-medium"
+              >
+                Back to Archives
+              </Link>
+              <button
+                onClick={this.handleReset}
+                className="px-4 py-2 border border-gray-300 text-gray-700 rounded hover:bg-gray-50 transition-colors font-medium"
+              >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)